databricks/koalas/frame.py

```diff
@@ -8331,6 +8331,92 @@ def quantile(self, q=0.5, axis=0, numeric_only=True, accuracy=10000):
 
         return DataFrame(internal) if not result_as_series else DataFrame(internal).T[key]
 
+    def query(self, expr, inplace=False):
+        """
+        Query the columns of a DataFrame with a boolean expression.
+
+        .. note::
+            * MultiIndex columns are not supported.
+            * If you want to use MultiIndex columns, you can just workaround
+              by using `DataFrame.apply` although it is less performance.
```

Comment:
`` `DataFrame.apply` `` -> `` :meth:`DataFrame.apply` ``. Can you also check the generated doc?

Reply:
`less performance` -> `less performant`